charts/perses/values.yaml

```diff
@@ -302,6 +302,11 @@ envVars: []
   # - name: PERSES_SECURITY_AUTHENTICATION_PROVIDERS_OIDC_1_CLIENT_SECRET
   #   value: "0815"
 
+# -- Automatically generate env vars for auth providers (client_id/client_secret) based on config.security.authentication.providers
+autoGenerateAuthEnvVars: true
```

Comment:
I'd try to follow same pattern we have on other charts like, wdyt?
```yaml
secret:
    create: true
    name: ""
```

If this makes sense we can deprecate the envVarsExternalSecretName I believ.

Reply:
I just leave it as deprecate so we can remove it later.